Program.cs

```diff
@@ -1,10 +1,24 @@
-﻿using Assignment;
+﻿using ChessBoardAssigment;
 
 class Program
 {
     static void Main()
     {
+        bool tryCodeAgain = true;
         ChessBoardDialog ChessBoardDialog = new();
```

Comment:
minor local variables naming convention -
![image](https://github.com/Pauliukas7/dotnet-console-app/assets/2812503/ebf643bd-8610-49ab-8fa5-587511f7b4af)

minor - for consistency if it's not an initialization value let's use var + type constructor 
so 
var tryCodeAgain = true;
var chessBoardDialog = new ChessBoardDialog(); etc.